include/ttmlir/Utils.h

```diff
@@ -127,6 +127,11 @@ inline MlirAttribute wrapArrayOfMlirAttributesAsAttribute(
   return wrap(mlir::ArrayAttr::get(unwrap(ctx), unwrappedAttributesArray));
 }
 
+// Checks if the given `mlir::Type` is a ranked tensor type.
+inline bool isRankedTensorType(mlir::Type type) {
```

Comment:
I would rather this function receive `Value` as one of the most general llvm classes which does have `getType()` member function that we can call inside here, it would make it more convenient for usage, because I think we are never dealing with types directly, so every call will be `isRankedTensorType(v.getType())`. In that case we might call this function `isRankedTensor` to make it less ambiguous about what it expects as an argument.